desktop/main/reactions/syncToRenderer.ts

```diff
@@ -217,6 +218,7 @@ export default (
       })
     ),
     $rootHash.pipe(map((rootHash) => ({ network: { rootHash } }))),
-    $currentLayer.pipe(map((currentLayer) => ({ network: { currentLayer } })))
+    $currentLayer.pipe(map((currentLayer) => ({ network: { currentLayer } }))),
+    $osPlatform.pipe(map(R.objOf('osPlatform')))
```

Comment:
If you think that this is the easiest way to push some data from the main process to the renderer you can do just `of({ osPlatform: os.platform() })` instead of having and passing subject ;)